play-ws-standalone/src/main/java/play/libs/ws/StandaloneWSRequest.java

```diff
@@ -32,7 +32,9 @@
      *
      * @return a promise to the response
      */
-    CompletionStage<? extends StandaloneWSResponse> get();
+    default CompletionStage<? extends StandaloneWSResponse> get() {
```

Comment:
Using default methods here will make this interface more simple and direct to implement.

The only doubt here is if `StandaloneAhcWSRequest` should instead return a `CompletionStage<StandaloneAhcWSResponse>`. It was currently return just like declared here in the interface.